api/logical.go

```diff
@@ -235,12 +236,13 @@ func (c *Logical) DeleteWithData(path string, data map[string][]string) (*Secret
 
 func (c *Logical) Unwrap(wrappingToken string) (*Secret, error) {
 	var data map[string]interface{}
+	wt := strings.TrimSuffix(string(wrappingToken), "\n")
```

Comment:
Would using `strings.TrimSpace` be better in here? We don't have to wrap `wrappingToken` around `string()` since it's already of that type.